src/components/Emails.js

```diff
@@ -104,6 +105,8 @@ class Emails extends Component {
           handleResend={this.props.handleResend}
           handleConfirm={this.props.handleConfirm}
           helpBlock={this.props.translate("emails.confirm_help_text")}
+          validationPattern={longCodePattern}
+          validationError={"modal.long_code_invalid_format"}
```

Comment:
How would this be affected by validation that has multiple possible errors to communicate to the user?